tests/test_integrated_channels/test_degreed2/test_client.py

```diff
@@ -160,6 +171,95 @@ def test_create_content_metadata_success(self):
         assert status_code == 200
         assert response_body == '"{}"'
 
+    @responses.activate
```

Comment:
I know it's redundant but could we also make sure to test the `_get` retry logic?